### PR TITLE
Fix plugin top center position

### PR DESCRIPTION
**Pull request by yaliscouto**
### 🏁 Goal
Position top center plugin panel correctly.

### 🖼 Screenshot

1 - After
plugin_top_center

2 - Before

Screenshot_1574712738

### ✅ How to test
1 - Create the following plugin:

```
class PluginCastButton(core: Core) : ButtonPlugin(core, name) {
    companion object : NamedType {
        override val name = "pluginCastButton"

        val entry = PluginEntry.Core(name = name, factory = { core -> PluginCastButton(core) })
    }

    override var panel: Panel = Panel.TOP

    override var position: Position = Position.CENTER

    override val resourceDrawable: Int
        get() = R.drawable.fullscreen_button

    override val idResourceDrawable: Int
        get() = R.id.fullscreen_button

    override val resourceLayout: Int
        get() = R.layout.button_plugin


    override fun onClick() {
        core.activePlayback?.stop()
    }
}
```

2 - Register this plugin in `BaseConfigPlugin`:
`Loader.register(PluginCastButton.entry)`

### CODE REVIEW

File: clappr/src/test/kotlin/io/clappr/player/plugin/control/MediaControlTest.kt [hunk 1]
```diff
@@ -125,9 +125,9 @@ class MediaControlTest {
 
         val plugin = core.plugins.asSequence().filterIsInstance(FakePlugin::class.java).first()
 
-        assertEquals(3, getTopCenterPanel().childCount, "Media Control Plugin should be added to Panel.TOP panel and Position.CENTER position in Media Control")
-        assertEquals(FakePlugin.viewId, getTopCenterPanel().getChildAt(2).id, "Media Control Plugin should be added to Media Control")
-        assertEquals(plugin.view, getTopCenterPanel().getChildAt(2))
+        assertEquals(1, getTopCenterPanel().childCount, "Media Control Plugin should be added to Panel.TOP panel and Position.CENTER position in Media Control")
+        assertEquals(FakePlugin.viewId, getTopCenterPanel().getChildAt(0).id, "Media Control Plugin should be added to Media Control")
+        assertEquals(plugin.view, getTopCenterPanel().getChildAt(0))
```
Comment (rcavaltw) [inline clappr/src/test/kotlin/io/clappr/player/plugin/control/MediaControlTest.kt]: You can replace these assertions with the existing helper

```kotlin
        assertMediaControlPanel(getTopCenterPanel(), Panel.TOP, Position.CENTER)
```

File: clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt [hunk 2]
```diff
@@ -74,7 +74,7 @@ open class MediaControl(core: Core, pluginName: String = name) :
 
     private val controlsPanel by lazy { view.findViewById(R.id.controls_panel) as RelativeLayout }
 
-    private val topCenterPanel by lazy { view.findViewById(R.id.top_center) as RelativeLayout }
+    private val topCenterPanel by lazy { view.findViewById(R.id.top_center) as LinearLayout }
```
Comment (rcavaltw) [inline clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt]: Using the more general interface `ViewGroup` should be enough in this context.

```suggestion
    private val topCenterPanel by lazy { view.findViewById<ViewGroup>(R.id.top_center) }
```

File: clappr/src/test/kotlin/io/clappr/player/plugin/control/MediaControlTest.kt (hunk 1 above)
Comment (tuliohmendes) [reply clappr/src/test/kotlin/io/clappr/player/plugin/control/MediaControlTest.kt]: Done 44a9d210

File: clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt (hunk 2 above)
Comment (tuliohmendes) [reply clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt]: Done 44a9d210

**Pull request merged**